### ISSUE TITLE
tvplayer.com plugin fails

### ISSUE THREAD

**Issue by whittinghamj** [streamlink/streamlink#3145]
## Bug Report

- [ x] This is a bug report and I have read the contribution guidelines.

### Description

Im using the following streamlink command.

streamlink --hls-live-restart --player-external-http --player-external-http-port '2223' --tvplayer-email 'user@gmail.com' --tvplayer-password 'hidden_for_github' --url 'https://tvplayer.com/watch/channel4' --default-stream '1080p,720p,480p,best'

### Expected / Actual behavior

stream to start

### Reproduction steps / Explicit stream URLs to test

1. ... run the above code using a valid username and password

### Log output

```
[cli][info] streamlink is running as root! Be careful!
[cli][debug] OS:         Linux-4.15.0-112-generic-x86_64-with-Ubuntu-18.04-bionic
[cli][debug] Python:     3.6.9
[cli][debug] Streamlink: 1.5.0
[cli][debug] Requests(2.18.4), Socks(1.6.5), Websocket(0.44.0)
[cli][info] Found matching plugin tvplayer for URL https://tvplayer.com/watch/bbctwo
[cli][debug] Plugin specific arguments:
[cli][debug]  --tvplayer-email=user@gmail.com (email)
[cli][debug]  --tvplayer-password=******** (password)
[plugin.tvplayer][debug] Logging in as user@gmail.com
[plugin.tvplayer][debug] Got stream attributes: {}
[plugin.tvplayer][debug] Missing 'expiry' from stream attributes
[plugin.tvplayer][debug] Missing 'key' from stream attributes
[plugin.tvplayer][debug] Missing 'token' from stream attributes
[plugin.tvplayer][debug] Missing 'uvid' from stream attributes
error: No playable streams found on this URL: https://tvplayer.com/watch/bbctwo
```

### Additional comments, screenshots, etc.

I think they might have changed something?

Here is an example of the missing vars pulled from their website

https://live.tvplayer.com/live1/itv1/bitrate1-clear.isml/fta.m3u8?key=2Pw1Eg0Px7Ey9Jm3Ly8Dr5Ci5Vc5Mk&platform=chrome&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9saXZlLnR2cGxheWVyLmNvbS9saXZlMS9pdHYxL2JpdHJhdGUxLWNsZWFyLmlzbWwvZnRhLm0zdTg~a2V5PTJQdzFFZzBQeDdFeTlKbTNMeThEcjVDaTVWYzVNayZwbGF0Zm9ybT1jaHJvbWUiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE1OTkxNzY2MzB9LCJJcEFkZHJlc3MiOnsiQVdTOlNvdXJjZUlwIjoiMTU5LjI0Mi4xMDQuMTYzLzMyIn19fV19&Expires=1599176630&Signature=QoIbYpaU1rvh8xfTP7IVnzr8H7xlRpe2VOs9xS07UHHvCvJuCCcZlqvNlbQJPSLNQG~6hGHNSbxRJM0TbMvrTd5bZdNpONA3jPiZZvImOD0UsPalIBVQx8T5k1lTcQF~6-SJEmHMhAUitWmvJPtkN55TA98MARSB8dQ4MW08Xx2uXdMvKQS4e2nccLeTE~lXzwtRU35MCPZzFrU3cwYuCnLV1UtNX7awabNSp5U1Nx9rCIqR8RtZ472QoNJvpMslb-YDuCPxuKUdMEgNDnpvQQxw4NqPZRdOR1Cq0R-GhpYY5WGm4r16wdZTfefuNWo4IPs9bziaJP-yULvjgkCdtg__&Key-Pair-Id=APKAJKZXHV4FL3DNMOCA

[Love Streamlink? Please consider supporting our collective. Thanks!](https://opencollective.com/streamlink/donate)

**Comment by mkbloke** [comment]
Interesting.  I cannot reproduce this.  I even tried with the wrong password and that still works!

I wonder if it's a phased site update being rolled out?

I see the tvplayer plugin has not changed since version 1.5.0 was released.

**Comment by mkbloke** [comment]
Actually, there is one way I can reproduce the same result, which is by trying to use the site from a non-UK IP address.  If you're using a UK IP address (as I assume you are likely to be doing), I guess it's also possible their IP geo-location service could have gone wonky.  It has done so before (see: #2110).

**Comment by whittinghamj** [comment]
I had not thought about the GEO lock - I will try on a UK server and get back to you on this.

**Comment by whittinghamj** [comment]
Ok this looks like a GEO lock issue - apologies !!!

[cli][info] streamlink is running as root! Be careful!
[cli][info] Found matching plugin tvplayer for URL https://tvplayer.com/watch/channel4
[cli][info] Available streams: 360p (worst), 432p, 504p, 576p (best)
[cli][info] Starting server, access with one of:
[cli][info]  http://127.0.0.1:2223/

**Comment by mkbloke** [comment]
Thanks for the update.

**Comment by whittinghamj** [comment]
I need to reopen this ticket / case.

I found a bug in the tvplayer plugin

The following URL does not work.

https://tvplayer.com/uk/watch/channel5

However the following URL DOES work

https://tvplayer.com/watch/channel5

When you look at the URL on their website though it gives you https://tvplayer.com/uk/watch/channel5 so streamlink breaks.

I've confirmed this several times so you just need to strip out the 'uk/' that the website gives you and streamlink works perfectly at this point.

Thanks
